packages/syft/src/syft/node/routes.py

```diff
@@ -123,7 +123,7 @@ def handle_login(email: str, password: str, node: AbstractNode) -> Response:
 
         if isinstance(result, SyftError):
             logger.bind(payload={"email": email}).error(result.message)
-            response = {"Error": result.message}
```

Comment:
@tcp @IonesioJunior , I am currently modifying the login endpoint to returning SyftError directly, to main consistency, would this affect the frontend PR, if a user logs in with wrong credentials.